src/ipyforcegraph/graphs.py

```diff
@@ -72,27 +72,19 @@ def _check_behavior_order(self, proposal: T.Bunch) -> Tuple[Behavior, ...]:
         """Ensure behaviors are not unwittingly being put in the wrong order."""
         behaviors: Tuple[Behavior, ...] = proposal.value
 
-        # TODO: Investigate making ordering a Behavior class variable
```

Comment:
again to my previous point: warnings and complex validators get swallowed. I think we strip this whole thing out, and implement the sorting on the typescript side, treating the happenstance order of the tuple as an implementation detail, since we can't serialize a set.